app/components/task_list_component/view.rb

```diff
@@ -14,9 +14,8 @@ def initialize(completed_task_count: nil, total_task_count: nil, sections: [], c
 
     def render_counter?
       return false unless @completed_task_count.present? && @total_task_count.present?
-      return true if FeatureService.enabled?(:draft_live_versioning)
 
-      @completed_task_count != @total_task_count
+      true
```

Comment:
The logic of this is a bit hard to follow, can we simplify it to just

```suggestion
      @completed_task_count.present? && @total_task_count.present?
```